libr/core/anal.c

```diff
@@ -255,6 +255,16 @@ R_API char *r_core_anal_fcn_autoname(RCore *core, ut64 addr, int dump) {
 				if (dump) {
 					r_cons_printf ("0x%08"PFMT64x" 0x%08"PFMT64x" %s\n", ref->at, ref->addr, f->name);
 				}
+				if (strstr (f->name, "__stack_chk_guard"))
```

Comment:
put this logic in a separate function, with an array and a loop.
if (blacklistedWord(f->name)) { break; }